src/main/scala/redis/RedisCluster.scala

```diff
@@ -83,6 +90,13 @@ case class RedisCluster(redisServers: Seq[RedisServer],
        getClusterSlots().map { clusterSlot =>
          log.info("refreshClusterSlots: " + clusterSlot.toString())
          clusterSlotsRef.single.set(Some(clusterSlot))
+         val serverSet = clusterSlot.keysIterator.map(_.master.hostAndPort).toSet
+         redisServerConnections.keys.foreach { server =>
+           if (!serverSet.contains(server)) {
+             redisServerConnections.remove(server)
+               .map(connection => _system.scheduler.scheduleOnce(1.second)(_system.stop(connection.actor)))
```

Comment:
Is it the correct way to disconnect from a cluster node? I tried to do it without scheduling, but failed [the clusterNodes test](https://github.com/etaty/rediscala/blob/master/src/test/scala/redis/RedisClusterTest.scala#L135)